app/assets/javascripts/ui_models/application.ts

```diff
@@ -103,7 +97,7 @@ export class WebApplication extends SNApplication {
     /** Allow our Angular directives to be destroyed and any pending digest cycles
      * to complete before destroying the global application instance and all its services */
     setImmediate(() => {
-      super.deinit();
+      super.deinit(source);
     })
```

Comment:
BTW setImmediate is [not standard](https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate) and if all you want is to run code on the next event cycle you can use setTimeout with no specified delay (defaults to 0)

Reply:
Oh interesting..